src/appimagetool/appdirtool.go

```diff
@@ -665,6 +666,7 @@ func handleGStreamer(appdir helpers.AppDir) {
 			} else {
 				log.Println("Bundling dependencies of GStreamer 1.0 directory...")
 				determineELFsInDirTree(appdir, locs[0])
+				determineELFsInDirTree(appdir, "/usr/lib64/gio/modules/libgiognutls.so")
```

Comment:
I fear that the `/usr/lib64/` is specific to your distribution. Fedora-type distributions tend to use `/usr/lib64/` whereas others tend to use `lib/x86_64-linux-gnu`...

Reply:
I agree, like I already wrote I would like your input on how you want to do it. I'm not a Go programmer so I'm learning as I work on this.
The only distro I've only seen use /usr/lib/x86_64-linux-gnu is debian and derivatives of debian (which most are basically debian or ubuntu variants as most debian distros use the same packaging).
Everyone else use /usr/lib or /usr/lib64, this is common for Redhat/Fedora/Centos, SUSE/openSUSE, Mageia, Arch and most other I've seen.

Reply:
> The only distro I've seen use /usr/lib/x86_64-linux-gnu is debian and derivatives of debian

...which makes the vast majority in terms of market share of Linux distributions.